R/descriptives.R

```diff
@@ -393,6 +395,7 @@ Descriptives <- function(jaspResults, dataset, options) {
   resultsCol[["Std. Error of Mean"]]      <- .descriptivesDescriptivesTable_subFunction_OptionChecker(options$standardErrorMean, na.omitted, function(param) { sd(param)/sqrt(length(param))} )
   resultsCol[["Median"]]                  <- .descriptivesDescriptivesTable_subFunction_OptionChecker(options$median,            na.omitted, median)
   resultsCol[["Std. Deviation"]]          <- .descriptivesDescriptivesTable_subFunction_OptionChecker(options$standardDeviation, na.omitted, sd)
+    resultsCol[["Coefficient of Variation"]]          <- .descriptivesDescriptivesTable_subFunction_OptionChecker(options$cOfVariation, na.omitted, function(param) { sd(param)/mean(param)} )
```

Comment:
same stuff about indentation here:
```suggestion
  resultsCol[["Coefficient of Variation"]]<- .descriptivesDescriptivesTable_subFunction_OptionChecker(options$cOfVariation,      na.omitted, function(param) { sd(param) / mean(param)})
```
so ideally, we have a space around most infix operators (`<-`, `+`, `-`, etc.) but in this case it's okay to forego that over the alignment.